spyder/app/cli_options.py

```diff
@@ -111,16 +111,6 @@ def get_options(argv=None):
               "confirmation messages and 'verbose' logs a "
               "lot of detailed information.")
     )
-    parser.add_argument(
-        '--debug-output',
-        default='terminal',
-        dest="debug_output",
-        choices=['terminal', 'file'],
-        help=("Print internal debugging info either to the "
-              "terminal or to a file called spyder-debug.log "
-              "in your current working directory. Default is "
-              "'terminal'.")
-    )
```

Comment:
Please revert this.

Reply:
I am unclear about what purpose this serves. The LSP logs are written to files if Spyder is in debug mode. Shouldn't Spyder debug files also be written if in debug mode? Is there an issue if debug files are always written if debug is enabled?

Reply:
I mentioned my reasons in my main review https://github.com/spyder-ide/spyder/pull/15629#pullrequestreview-666225912 above:

> it makes uploading our debug file to Github when starting Spyder from a terminal harder than before. That's something I'd like to preserve for Anaconda users, our main distribution channel by far. Besides, outputting to a debug file from the command line is also valuable when Spyder can't be started at all.

Reply:
I'm sorry, I remember what you said before, I just don't completely understand it. I'm happy to revert it; I just want to understand better the points you made.
How does it make uploading the debug file to Github harder? Previously one had to use both `--debug-info=[minimal|verbose]` and `--debug-output=file`. With this they only need `--debug-info=[minimal|verbose]` (the file will be created if in debug mode).
How does this affect the situation when Spyder can't be started at all? Again, the file is create by virtue of the debug mode; it doesn't require changing anything while the app is running.

To be clear, this doesn't preclude creating a debug file from the commandline; it just no longer requires `--debug-output=file` to create it.

Reply:
> How does it make uploading the debug file to Github harder?

Because you have to find the debug file in our config directory, instead of getting it from the current working directory. I find that very convenient, especially on Linux where our config directory is hidden.

> To be clear, this doesn't preclude creating a debug file from the commandline; it just no longer requires --debug-output=file to create it.

Ohhh, ok, I understand your point now. So is it possible to have both cases? I mean, when running from the command line to generate the file in the current working directory and when using the Tools menu entry to save it in our config dir?

Reply:
Okay, I see what you mean now. This currently only sets one location for the debug file. I'll change it to what you describe.
* I'll revert the commandline option
* When the commandline option is used, then the debug file will go to the current directory.